SU-HKKU/cardinal_service_profile
# CSD-359 Updated related opportunity view and styles

# READY FOR REVIEW

# Summary
- Modified cta button within related opportunity view and updated styles

# Needed By (Date)
- 08/27

# Urgency
- medium

# Steps to Test

1. Pull this branch
2. Clear cache
3. View any individual opportunities page.
The title and button should now be centered.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [CSD-359](https://stanfordits.atlassian.net/browse/CSD-359)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
